src/sidebar/services/import-annotations.ts

```diff
@@ -128,6 +128,7 @@ export class ImportAnnotationsService {
     this._store.beginImport(anns.length);
 
     const existingAnns = this._store.allAnnotations();
+    const currentUri = this._store.mainFrame()?.uri;
```

Comment:
I can't remember whether `mainFrame` will work in the context of VitalSource books. In that context there is only one book chapter frame connected at a time, but it is not the host frame.

Reply:
Will try